aif360/metrics/utils.py

```diff
@@ -126,6 +120,29 @@ def compute_num_TF_PN(X, y_true, y_pred, w, feature_names, favorable_label,
         FN=np.sum(w[np.logical_and(y_true_pos, y_pred_neg)], dtype=np.float64)
     )
 
+def compute_ROC(y_true, scores, pos_label = 1, condition = None):
+
+    """Compute area under Receiver Operating Characteristic curve.
+
+    Args:
+        y_true (numpy.ndarray): True label vector.
+        scores (numpy.ndarray): Models confidences
+        condition (list(dict)): Same format as
+            :func:`compute_boolean_conditioning_vector`.
+
+    Returns:
+        False negative rate matrix for given classification thresholds (list)
+        True positive rate matrix for given classification thresholds  (list)
+        Area under Receiver Operating Characteristic Curve (float)
+    """
+
+    fpr, tpr, thresholds = metrics.roc_curve(y_true, scores, pos_label = pos_label)
```

Comment:
this should also take `instance_weights` (`sample_weight` in sklearn) into account.

it probably should use `roc_auc_score` but that doesn't seem to support the `pos_label` arg?

also, you're missing `sklearn` before `metrics`